diff --git a/node_modules/pathfinding/src/core/Util.js b/node_modules/pathfinding/src/core/Util.js
index ed7fd0f..f15cfc0 100644
--- a/node_modules/pathfinding/src/core/Util.js
+++ b/node_modules/pathfinding/src/core/Util.js
@@ -164,7 +164,7 @@ function smoothenPath(grid, path) {
             }
         }
         if (blocked) {
-            lastValidCoord = path[i - 1];
+            var lastValidCoord = path[i - 1];
             newPath.push(lastValidCoord);
             sx = lastValidCoord[0];
             sy = lastValidCoord[1];
